doc/connectivity/bluetooth/api/audio/bluetooth-le-audio-arch.rst

```diff
@@ -59,6 +59,333 @@ GAF has been implemented in Zephyr with the following structure.
+   | VCP Volume Controller      |          |           | C.6       |
+   +----------------------------+----------+-----------+-----------+
+   | VCP Volume Renderer        | O        |           |           |
+   +----------------------------+----------+-----------+-----------+
+   | MICP Microphone Controller |          |           | C.6       |
+   +----------------------------+----------+-----------+-----------+
+   | MICP Microphone Device     | O        |           |           |
+   +----------------------------+----------+-----------+-----------+
+   | CCP Call Control Server    |          | O         |           |
+   +----------------------------+----------+-----------+-----------+
+   | CCP Call Control Client    | O        |           | C.6       |
+   +----------------------------+----------+-----------+-----------+
+   | MCP Media Control Server   |          | O         |           |
+   +----------------------------+----------+-----------+-----------+
+   | MCP Media Control Client   | O        |           | C.6       |
+   +----------------------------+----------+-----------+-----------+
+   | CSIP Set Coordinator       |          | C.5       | C.7       |
+   +----------------------------+----------+-----------+-----------+
+   | CSIP Set Member            | C.7      |           |           |
+   +----------------------------+----------+-----------+-----------+
+
+Notes:
+
+* C.1: Support at least one of BAP Unicast Client or BAP Broadcast Source
+* C.2: Support at least one of BAP Unicast Server or BAP Broadcast Sink
+* C.3: Mandatory if BAP Broadcast Sink
+* C.4: Mandatory if BAP Scan Delegator
+* C.5: Mandatory if BAP Unicast Client
+* C.6: Support at least one
+* C.7: Mandatory if part of a coordinated set
+* C.8: Mandatory if the Commander transmits CAP announcements
+
+
+.. table:: CSIP dependencies
+   :widths: auto
+   :align: center
+
+   +------------+------------+-----------------+
+   |            | Set Member | Set Coordinator |
+   +============+============+=================+
+   | CSIS Server| M          |                 |
+   +------------+------------+-----------------+
+   | CSIS Client|            | M               |
+   +------------+------------+-----------------+
+
+
+The dependencies of the higher level profiles (GMAP, HAP, PBP and TMAP) are listed in the following tables.
+
+.. table:: GMAP dependencies
+   :widths: auto
+   :align: center
+
+   +----------------------------+----------------------+-----------------------+-----------------------+-------------------------+
+   |                            | Unicast Game Gateway | Unicast Game Terminal | Broadcast Game Sender | Broadcast Game Receiver |
+   +============================+======================+=======================+=======================+=========================+
+   | GMAS Server                | M                    |                       | M                     |                         |
```

Comment:
GMAS is not mandatory for the BGS if it's the only GMAP role enabled

It is, however mandatory for the BGR

Reply:
Can you point me to where in the spec this is specified?

Reply:
Section 3.8 in GMAP: 

> UGG, UGT, and BGR devices supporting GMAP shall implement the GATT Server role and instantiate only one GMAS. This is referred to as the GMA Server.
> BGS-only devices are not required by GMAP to implement the GATT Server role. Any BGS device that does implement the GATT Server role shall instantiate only one GMAS.

So if the BGS device does not implement the GATT server role, then it isn't required to have GMAS